.github/workflows/checks-generator.yaml

```diff
@@ -14,11 +18,12 @@ jobs:
     strategy:
       matrix:
         package:
-          - 'firebase-rules-coverage'
           - 'firebase-rules-generator'
         node_version:
           - '12'
```

Comment:
should we remove 12 then?

Reply:
This is the coverage package and this file is about generator. Because they are two packages, I will treat them in separate PRs and in the following package I will remove unsupported node versions. As you can see https://github.com/simpleclub/firebase-rules-helper/pull/22/files#diff-5c148a23087f21b63c00dbf193fe76a01cdfc09e6d523ae74e6817f387a2ddb5R25-R26 for the coverage package I've already done it.